aten/src/ATen/cuda/CUDAUtils.h

```diff
@@ -17,4 +17,18 @@ inline bool check_device(ArrayRef<Tensor> ts) {
   return true;
 }
 
+struct NoTF32Guard {
+#if defined(CUDA_VERSION) && CUDA_VERSION >= 11000
+  cublasHandle_t &handle;
+  cublasMath_t original_mode;
+  NoTF32Guard(cublasHandle_t &handle): handle(handle) {
```

Comment:
Same nit here. The ampersand describes the type.

Reply:
fixed